src/Microsoft.Health.Dicom.Core/Extensions/DicomDatasetExtensions.cs

```diff
@@ -62,13 +88,43 @@ public static T GetSingleValueOrDefault<T>(this DicomDataset dicomDataset, Dicom
         /// <param name="dicomDataset">The dataset to get the VR value from.</param>
         /// <param name="dicomTag">The DICOM tag.</param>
         /// <param name="expectedVR">Expected VR of the element.</param>
-        /// /// <remarks>If expectedVR is provided, and not match, will return null.</remarks>
-        /// <returns>An instance of <see cref="DateTime"/> if the value exists and comforms to the DA format; otherwise <c>null</c>.</returns>
-        public static DateTime? GetStringDateAsDate(this DicomDataset dicomDataset, DicomTag dicomTag, DicomVR expectedVR = null)
+        /// <remarks>If expectedVR is provided, and not match, will return null.</remarks>
+        /// <returns>An instance of <see cref="DateTimeOffset"/> if the value exists and comforms to the DA format; otherwise <c>null</c>.</returns>
+        public static DateTimeOffset? GetStringDateAsDate(this DicomDataset dicomDataset, DicomTag dicomTag, DicomVR expectedVR = null)
         {
             EnsureArg.IsNotNull(dicomDataset, nameof(dicomDataset));
             string stringDate = dicomDataset.GetSingleValueOrDefault<string>(dicomTag, expectedVR: expectedVR);
-            return DateTime.TryParseExact(stringDate, DateFormatDA, CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTime result) ? result : null;
+            return DateTimeOffset.TryParseExact(stringDate, DateFormatDA, CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTimeOffset result) ? result : null;
+        }
+
+        /// <summary>
+        /// Gets the DT VR value as <see cref="DateTime"/>.
+        /// </summary>
+        /// <param name="dicomDataset">The dataset to get the VR value from.</param>
+        /// <param name="dicomTag">The DICOM tag.</param>
+        /// <param name="expectedVR">Expected VR of the element.</param>
+        /// <remarks>If expectedVR is provided, and not match, will return null.</remarks>
+        /// <returns>An instance of <see cref="DateTime"/> if the value exists and conforms to the DT format; othewise <c>null</c>.</returns>
+        public static DateTimeOffset? GetStringDateTimeAsDateTimeOffset(this DicomDataset dicomDataset, DicomTag dicomTag, DicomVR expectedVR = null)
+        {
+            EnsureArg.IsNotNull(dicomDataset, nameof(dicomDataset));
+            string stringDateTime = dicomDataset.GetSingleValueOrDefault<string>(dicomTag, expectedVR: expectedVR);
+            return DateTimeOffset.TryParseExact(stringDateTime, DateTimeFormatsDT, CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTimeOffset result) ? result : null;
```

Comment:
Is this current? Should you be looking at the http://dicom.nema.org/medical/Dicom/2016e/output/chtml/part03/sect_C.12.5.html that tag to confirm the current culture?

You need to parse 2 DTs, one as is, put in TagValue and another in UTC needs you to config the Offset tag and if specified Get the UTC DT and use that in STOW

Reply:
Updated. Now we make two calls, one for getting the local time and one for UTC time.